diagnostics-app/src/app/diagnostics/diagnostics-dashboard/diagnostics-dashboard.component.html

```diff
@@ -0,0 +1,7 @@
+<!--
+ Copyright 2023 The Chromium Authors. All rights reserved.
+ Use of this source code is governed by a BSD-style license that can be
+ found in the LICENSE file.
+-->
+
+<app-battery></app-battery>
```

Comment:
Why do we use app-battery here?

I think we can have empty HTML like the event page.

Reply:
Done.